scu-it/test/fiskaltrust.Middleware.SCU.IT.AcceptanceTests/EpsonRTPrinterTests.cs

```diff
@@ -6,7 +6,7 @@ namespace fiskaltrust.Middleware.SCU.IT.AcceptanceTests
 {
     public class EpsonRTPrinterTests : ITSSCDTests
     {
-        private static readonly Uri _serverUri = new Uri("http://192.168.0.14/");
+        private static readonly Uri _serverUri = new Uri("http://194.93.177.143:9000");
```

Comment:
IDK if you want to share the IP address 🤷

Reply:
Ehm .. no thanks good catch :D 

```suggestion
        private static readonly Uri _serverUri = new Uri("http://192.168.0.14/");
```